core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -169,8 +169,10 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     } else {
       for {
         // we exclude workflows submitted by users that have exceeded the max workflows per user
-        excludedUsers <- dataAccess.workflowQuery.listSubmittersWithMoreWorkflowsThan(maxActiveWorkflowsPerUser, WorkflowStatuses.runningStatuses)
-        workflowRecs <- dataAccess.workflowQuery.findQueuedWorkflows(excludedUsers.map { case (submitter, count) => submitter }, SubmissionStatuses.terminalStatuses :+ SubmissionStatuses.Aborting).take(batchSize).result
+        excludedUsersMap <- dataAccess.workflowQuery.listSubmittersWithMoreWorkflowsThan(maxActiveWorkflowsPerUser, WorkflowStatuses.runningStatuses).map(_.toMap)
```

Comment:
TOL a `.map` within a for-comp can often be more neatly expressed as a separate line (or in this case, perhaps combined with the line below?)

Reply:
I felt in this case that a mapping of `toMap` was pretty trivial and didn't warrant its own line like this:
```
excludedUsersTuples <- dataAccess.workflowQuery.listSubmittersWithMoreWorkflowsThan(maxActiveWorkflowsPerUser, WorkflowStatuses.runningStatuses)
excludedUsersMap = excludedUsersTuples.toMap
excludedUsers = excludedUsersMap.keys.toSeq
```